OFL.txt

```diff
@@ -1,4 +1,4 @@
-Copyright 20** The Unified Font Repository Project Authors (https://github.com/googlefonts/Unified-Font-Repository)
+Copyright 20** My Font Project Authors (https://github.com/googlefonts/My-Font-Repository)
```

Comment:
I am actually not sure if this should be the license of the UFR, or the actual template for the font. Since Fontbakery require consistency between the font info and the OFL.txt I chose the latest.

Reply:
The license files we need (OFL) will be for the fonts in the repository created from this template, not for the repo itself. So this change is correct.